e2e/app/google-login.ts

```diff
@@ -91,11 +91,13 @@ export default class GoogleLoginPage extends BasePage {
    */
   async load(): Promise<void> {
     const url = configs.uiBaseUrl + configs.loginUrlPath;
-    await this.page.goto(url, {waitUntil: ['networkidle0', 'domcontentloaded'], timeout: 30000}).catch((err) => {
+    try {
+      await this.page.goto(url, {waitUntil: ['networkidle0', 'domcontentloaded'], timeout: 30000});
+    } catch (err) {
       console.error('Google login page not found. ' + err);
-      this.takeScreenshot('GoogleLoginNotFound');
+      await this.takeScreenshot('GoogleLoginNotFound');
```

Comment:
nit: maybe specify `GoogleLoginPageNotFound` to distinguish from the case of an unrecognized user.